spec/lib/scrabble_spec.rb

```diff
@@ -0,0 +1,50 @@
+require './lib/scrabble/scrabble.rb'
+
+describe Scrabble::Scrabble do
```

Comment:
Hmm maybe I am missing something.  Lets take a couple minutes in class and walk me through your test cases.   I feel like we are reaching the limits of what can be conveyed through github messages.